docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1098,6 +1099,121 @@ team can more easilly read the messages.
+- fhir-bulkimportexport-webapp   
+
+To integration test BulkData, there are tests in ExportOperationTest.java in fhir-server-test module with server integration test cases for system, patient and group export.  
+The *fhir-operation-bulkdata* project implements the REST APIs for Bulkdata export as FHIR operations.  There are three operations ExportOperation - system export, PatientExportOperation - Patient export and GroupExportOperation - group export.  Each operation calls the JavaBatch framework defined in the *fhir-bulkimportexport-webapp* project to execute the export unit-of-work.   
+There are 3 chunk style JavaBatch jobs defined as following in *fhir-bulkimportexport-webapp* project for the above 3 export operations:  
+
+- FhirBulkExportChunkJob
+- FhirBulkExportPatientChunkJob
+- FhirBulkExportGroupChunkJob
+
+The *fhir-bulkimportexport-webapp* module is a wrapper for the whole BulkData web application, which is the build artifact - fhir-bulkimportexport.war.   This web archive is copied to the apps directory of the liberty fhir-server instance. Following is a sample liberty server configuration (server.xml) for fhir-bulkimportexport.war:
+
+```xml
+    <webApplication id="fhir-bulkimportexport-webapp" location="fhir-bulkimportexport.war" name="fhir-bulkimportexport-webapp">
+        <classloader commonLibraryRef="fhirSharedLib" privateLibraryRef="configResources,fhirUserLib"/>
+        <application-bnd>
+            <security-role id="users" name="FHIRUsers">
+                <group name="FHIRUsers"/>
+            </security-role>
+        </application-bnd>
+    </webApplication>
+```
+
+BulkData web application writes the exported FHIR resources to IBM COS or Amazon S3 bucket, as configured in the per-tenant bulkdata.json configuration file.  The bulkdata.json file is stored in the tenant configuration directory of each fhir-server instance. The following is a bulkdata.json which is configured to export the FHIR resources into fhir-bulkdata-sample bucket of IBM COS: 
+
+```json
+{
+  "applicationName": "fhir-bulkimportexport-webapp",
+  "moduleName": "fhir-bulkimportexport.war",
+  "jobParameters": {
+    "cos.bucket.name": "fhir-bulkdata-sample",
+    "cos.location": "us",
+    "cos.endpointurl": "https://s3.us-south.cloud-object-storage.appdomain.cloud",
+    "cos.credential.ibm": "Y",
+    "cos.api.key": "xxxxxxxxxxxxxxxxxxxxxxx",
+    "cos.srvinst.id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+  },
+  "implementation_type": "cos",
+  "batch-uri": "https://localhost:9443/ibm/api/batch/jobinstances",
+  "batch-user": "fhiradmin",
+  "batch-user-password": "xxxxxxxxxxxx",
+  "batch-truststore" : "resources/security/fhirTruststore.jks",
+  "batch-truststore-password" : "xxxxxxxxxxxx",
+  "serverHostname" : "localhost:9443",	       <!--Host name part of the server generated polling location url -->
+  "contextRoot" : "/fhir-server/api/v4"		   <!--Context root part of the server generated polling location url -->
+}
```

Comment:
```
<!--Host name part of the server generated polling location url -->
  "contextRoot" : "/fhir-server/api/v4"		   <!--Context root part of the server generated polling location url -->
}
```
I think these comments should be outside of the JSON

Reply:
yeah, make sense

Reply:
done